src/Icon/README.md

```diff
@@ -0,0 +1,36 @@
+# Icon
+To implement Icon component into your project you'll need to check your icon in the [list of all icons](../icons/icons.md) (*if you don't know the name*) and add an import of the icon name you want to use e.g.:
+```jsx
+import Airplane from "@kiwicom/orbit-components/lib/icons/Airplane";
+```
+or if you want to import more icons at once:
+```jsx
+import { Airplane, ArrowUp, ChevronDown } from "@kiwicom/orbit-components/lib/icons";
+```
+After adding import into your project you can use it simply like:
+```jsx
+<Airplane />
+<ArrowUp />
+<ChevronDown />
+```
+## :rocket: Props
+Table below contains all type of the props available in Icon component.
+
+| Name          | Type            | Default         | Description                      |
+| :------------ | :-------------- | :-------------- | :------------------------------- |
+| className     | `string`        |                 | Pass className.
+| color         | [`enum`](#enum) | `currentColor`  | The color of the Icon.
+| customColor   | `string`        |                 | The customColor of the Icon. [See Functional specs](#warning-functional-specs) 👀
+| **size**      | [`enum`](#enum) | `"medium"`      | The size of the Icon.
+
+### enum
+
+| color         | size       |
+| :------------ | :--------- |
+| `"attention"` | `"small"`  |
+| `"primary"`   | `"medium"` |
+| `"secondary"` | `"large"`  |
+| `"terciary"`  |            |
+
+## :warning:️ Functional specs
+* By passing the `customColor` prop into Icon, it will overwrite `color` prop for custom color chosen by user.
```

Comment:
I would delete `for custom color chosen by user`.
Also I would add something like:
If you don't pass `customColor` or `color` to icon, it will inherit color from parent container with`currentColor`.